internal/resource/http.go

```diff
@@ -247,3 +253,21 @@ func (c HttpClient) getReaderWithHeader(url string, header http.Header) (io.Read
 		}
 	}
 }
+
+// addSystemInfoHeaders will add an ID, a VERSION_ID (if present), and a
+// VARIANT_ID (if present) header to the client's ignition config GET request
+func addSystemInfoHeaders(header *http.Header) error {
+	or, err := osrelease.New()
+	if err != nil {
+		return err
+	}
+
+	header.Add("ID", or.ID)
+	if len(or.VERSION_ID) != 0 { // optional field
+		header.Add("VERSION_ID", or.VERSION_ID)
+	}
+	if len(or.VARIANT_ID) != 0 { // optional field
+		header.Add("VARIENT_ID", or.VARIANT_ID)
```

Comment:
A typo here and in multiple other places in this PR, it's "VARI**A**NT" with "A".

Reply:
Fixed